kernel/base/src/main/java/com/twosigma/beakerx/kernel/commands/MagicCommand.java

```diff
@@ -178,45 +183,80 @@ private MagicCommandFunctionality classpathAddJar() {
           throw new IllegalStateException("Wrong command format: " + CLASSPATH_ADD_JAR);
         }
 
-        this.kernel.addJarToClasspath(new PathToJar(split[3]));
+        String path = split[3];
+        if (doesPathContainsWildCards(path)) {
+          getPaths(path)
+              .forEach(currentPath -> kernel.addJarToClasspath(new PathToJar(currentPath)));
```

Comment:
so we are expanding wildcard instead of letting java handle it.
i think this could cause a problem on windows since there is a fairly small limit to total size of the classpath.
otoh i like the ability to report exactly the matched files.